pkg/cluster/manager.go

```diff
@@ -307,11 +307,12 @@ func (m *Manager) CleanCluster(clusterName string, gOpt operator.Options, cleanO
 			target = "log"
 		}
 		if err := cliutil.PromptForConfirmOrAbortError(
-			"This operation will clean %s %s cluster %s's %s.\nNodes will be ignored: %s\nRoles will be ignored: %s\nDo you want to continue? [y/N]:",
+			"This operation will clean %s %s cluster %s's %s.\nNodes will be included:%s\nNodes will be ignored: %s\nRoles will be ignored: %s\nDo you want to continue? [y/N]:",
```

Comment:
IMO, if one specified `-n`, then ONLY those nodes will be cleaned, this is the opposite condition to `--ignore-node`,  so this prompt message should be an independent one, such as:

```bash
This operation will clean the following nodes from cluster xxx:
172.10.1.0:4000
172.10.1.1:4000
Do you want to continue? [y/N]:"
```

Reply:
@anywhy @9547 I think we can refactor this message as:

```
This operation will clean the following directories from cluster xxx:
172.10.1.0:/path/to/dir1
172.10.1.0:/path/to/dir2
172.10.1.1:/path/to/dir
```

No matter the user specify `-N` or `--ignore-node`, we can use the same message.

This will make the clean action more transparent.

Reply:
I think this could be a separate PR, otherwise this one would be too large